app/helpers/plantings_helper.rb

```diff
@@ -40,4 +40,18 @@ def display_planting(planting)
       "#{planting.owner}."
     end
   end
+
+  def plantings_active_tickbox_path(owner, show_all)
+    show_inactive_tickbox_path('plantings', owner, show_all)
+  end
+
+  def plantings_title(owner, crop)
```

Comment:
I suppose this could be unified with `harvests_title`, but I don't think it's worth it yet. Maybe if we add another similar helper (`posts_title`?) then we could unify all three.